.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: "Integration Tests"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - id: upload
+      uses: ./
+      with:
+        source_dir: tests/files
+        container_name: test
+        connection_string: '${{ secrets.AZURE_STORAGE_CONNECTION_STRING }}'
```

Comment:
You'll need to setup a real test blob container to run this integration test.
Maybe makes more sense run this with some kind of emulation like Azurite